classes/class-gf-securesubmit.php

```diff
@@ -707,6 +753,15 @@ public function scripts()
                 ),
             ),
             array(
+                'handle' => 'songbird.js',
+                'src' => 'https://includestest.ccdc02.com/cardinalcruise/v1/songbird.js',
```

Comment:
Does this URL work for prod and test? If so, it might be nice to remove the manual check in our WooCommerce and Magento integrations

Reply:
nah. i need to check for this.